lib/vehicle_factory.rb

```diff
@@ -0,0 +1,19 @@
+class VehicleFactory
+
+  def create_vehicles(registrations)
+    vehicles = []
+    registrations.each do |registration|
+      vehicles << Vehicle.new({
+        vin: registration[:vin_1_10], 
+        year: registration[:model_year], 
+        model: registration[:model],
+        make: registration[:make],
+        engine: :ev})
+    end
+    vehicles
+  end
```

Comment:
Nice! This could also be a good use of `map`. That way, you wouldn't need a placeholder on line 4, and you wouldn't need to return `vehicles` on line 13.